modules/test/module-configuration-localization-test/src/testIntegration/java/com/liferay/module/configuration/localization/test/ModuleConfigurationLocalizationTest.java

```diff
@@ -93,9 +100,7 @@ private String _collectBundleError(Bundle bundle) {
 		Collections.addAll(pids, extendedMetaTypeInformation.getFactoryPids());
 		Collections.addAll(pids, extendedMetaTypeInformation.getPids());
 
-		String bundleName = bundle.getSymbolicName();
-
-		if (pids.isEmpty() || !bundleName.startsWith("com.liferay")) {
+		if (pids.isEmpty() || !_checkPIDScope(pids)) {
 			return StringPool.BLANK;
 		}
```

Comment:
@AkramYoussoufi, the current fix is not right. I noticed the logic is for the bundle `com.liferay.portal.vulcan.impl`. Its pids are as the following:
`[com.liferay.portal.vulcan.internal.configuration.VulcanCompanyConfiguration, com.liferay.portal.vulcan.internal.configuration.VulcanConfiguration, graphql.kickstart.servlet.OsgiGraphQLHttpServlet, com.liferay.portal.vulcan.internal.configuration.HeadlessAPICompanyConfiguration]`

The current fix will skip checking for  `VulcanCompanyConfiguration`, `VulcanConfiguration` and `HeadlessAPICompanyConfiguration`.

You may add the logic in the while block.

```
while (iterator.hasNext()) {
			String pid = iterator.next();

			if (!pid.startsWith("com.liferay")) {
				iterator.remove();

				continue;
			}
	
			ExtendedObjectClassDefinition extendedObjectClassDefinition =
				extendedMetaTypeInformation.getObjectClassDefinition(
					pid, locale.getLanguage());

			if (!_isGenerateUI(extendedObjectClassDefinition)) {
				iterator.remove();
			}
		}
```